store/tikv/coprocessor.go

```diff
@@ -1268,6 +1279,12 @@ func (it *copIterator) Close() error {
 	it.rpcCancel.CancelAll()
 	it.actionOnExceed.close()
 	it.wg.Wait()
+	failpoint.Inject("ticase-4170", func(val failpoint.Value) {
```

Comment:
Will it be better if we put this in `copIterator.Next`

``` go
func (it *copIterator) Next(ctx context.Context) (kv.ResultSubset, error) {
    defer func(){
        if resp == nil{
            failpoint.Inject
        }
    }()
```

Reply:
good advice, updated.